### DISCUSSION TITLE
hi i want to prevent imask from reseting the cursor to the cursor positon 0 when i click on a section of the mask. you can see what i am referrring to if you click on m in the lazy mode example here.  https://imask.js.org/guide.html#masked-date

### DISCUSSION THREAD

**Discussion by Hestre** [uNmAnNeR/imaskjs#971] [Q&A]
hi i want to prevent imask from reseting the cursor to the cursor positon 0 when i click on a section of the mask. you can see what i am referrring to if you click on m in the lazy mode example here.  https://imask.js.org/guide.html#masked-date

_Originally posted by @Hestre in https://github.com/uNmAnNeR/imaskjs/issues/467#issuecomment-1827865733_

**Comment by uNmAnNeR** [comment]
hi. there is no flag for it but you can overwrite [nearestInputPos](https://imask.js.org/api/classes/imask.Masked.html#nearestInputPos) method in your mask
